dist/mapbox-gl.css

```diff
@@ -1,433 +1,556 @@
 }
 
 .mapboxgl-ctrl.mapboxgl-ctrl-attrib {
-    padding: 0 5px;
-    background-color: rgba(255, 255, 255, .5);
-    margin: 0;
+  padding: 0 5px;
+  background-color: rgba(255, 255, 255, 0.5);
+  margin: 0;
 }
+
 .mapboxgl-ctrl-attrib.mapboxgl-compact {
-    padding-top: 2px;
-    padding-bottom: 2px;
-    margin: 0 10px 10px 10px;
-    position: relative;
-    padding-right: 24px;
-    background-color: #fff;
-    border-radius: 3px 12px 12px 3px;
-    visibility: hidden;
+  padding-top: 2px;
+  padding-bottom: 2px;
+  margin: 0 10px 10px;
+  position: relative;
+  padding-right: 24px;
+  background-color: #fff;
+  border-radius: 3px 12px 12px 3px;
+  visibility: hidden;
 }
+
 .mapboxgl-ctrl-attrib.mapboxgl-compact:hover {
-    visibility: visible;
-}
-.mapboxgl-ctrl-attrib.mapboxgl-compact:after {
-    content: '';
-    cursor: pointer;
-    position: absolute;
-    bottom: 0;
-    right: 0;
-    background-image: url("data:image/svg+xml;charset=utf-8,%3Csvg%20viewBox%3D%270%200%2020%2020%27%20xmlns%3D%27http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%27%3E%0D%0A%09%3Cpath%20fill%3D%27%23333333%27%20fill-rule%3D%27evenodd%27%20d%3D%27M4%2C10a6%2C6%200%201%2C0%2012%2C0a6%2C6%200%201%2C0%20-12%2C0%20M9%2C7a1%2C1%200%201%2C0%202%2C0a1%2C1%200%201%2C0%20-2%2C0%20M9%2C10a1%2C1%200%201%2C1%202%2C0l0%2C3a1%2C1%200%201%2C1%20-2%2C0%27%20%2F%3E%0D%0A%3C%2Fsvg%3E");
-    background-color: rgba(255, 255, 255, .5);
-    width: 24px;
-    h
```

Comment:
I don't like the idea of mixing these all together, since they are unrelated, and just happen to have the same values.

It also mixes up the stylesheet document, instead of having attribution styling together and then user-location-dot styling together it's now mixed.